collector/lib/Process.cpp

```diff
@@ -23,29 +23,143 @@ You should have received a copy of the GNU General Public License along with thi
 
 #include "Process.h"
 
+#include <chrono>
+
+#include "CollectorStats.h"
+#include "SysdigService.h"
+
 namespace collector {
 
+const std::string Process::NOT_AVAILABLE("N/A");
```

Comment:
I think something longer and more descriptive might be better. E.g "Process could not be determined by Collector"